procedures/unit-testing-basics.ipf

```diff
@@ -733,6 +736,32 @@ static Function CreateTestRunSetup(procWinList, matchStr, enableRegExp, errMsg,
 	endfor
 	Redimension/N=(tdIndex, -1, -1, -1) testRunData
 
+	if(hasDGen)
+		UTF_Test_MD_Gen#ExecuteAllDataGenerators()
+	endif
+
+	for(i = 0; i < tdIndex; i += 1)
+		dgenList = testRunData[i][%DGENLIST]
+
+		if(UTF_Utils#IsEmpty(dgenList))
+			continue
+		endif
+
+		procWin = testRunData[i][%PROCWIN]
+		fullFuncName = testRunData[i][%FULLFUNCNAME]
+
+		if(UTF_Test_MD#CheckFunctionSignatureTC(procWin, fullFuncName, markSkip))
```

Comment:
I think we should change the old behavior of skipping the TC in favor of aborting, though I am open to discuss that as I might miss an application for this behavior.

@t-b what do you think?

Reply:
@MichaelHuth 
> I think we should change the old behavior of skipping the TC in favor of aborting, though I am open to discuss that as I might miss an application for this behavior.

Can you elaborate?

Reply:
I was wondering if this behavior is used in some real world case.
Because with the skip behavior an accidental wrongly assigned data generator to an test case where the signature does not fit results in silently skipping that test case.
I would rate that as a wrong test setup, where the IUTF should abort.

Reply:
> I would rate that as a wrong test setup, where the IUTF should abort.

Totally agree!

Reply:
I cannot remove the skip in `CheckFunctionSignatureTC` completely. `CheckDataGenZeroSize` is setting the test case as skip if the data generator returned an empty wave (which is not an error).

Reply:
All other conditions for setting test cases are now removed and changed into an abort.